draft-vanbrouwershaven-acme-auto-discovery.md

```diff
@@ -434,13 +456,13 @@ One potential security risk associated with the mechanism defined in this docume
 
 To minimize this risk, ACME clients must be written with a cautious and security-conscious approach when interacting with ACME servers. It is crucial not to blindly trust servers to behave securely and in accordance with the ACME protocol.
 
-## ACME Keys
+## Authorization of ACME Keys
 
-To ensure a secure account binding per customer, it is essential that each customer possesses their own unique ACME key. The utilization of individual ACME keys allows for a distinct association between the customer's account and the established account binding.
+To ensure a secure account binding, it is essential that each customer be able to control and pre-approve the ACME account keys that are authorized to request certificates against their account, particularly for certificate types that have billing implications, or that have additional verification requirements beyond what can be done inline within the ACME protocol, such as OV, EV [[EVBRs]], Qualified Website [[QWAC-ENISA]], or Verified Mark Certificates [[VMCRequirements]].
```

Comment:
Inline with the suggestion above:

```suggestion
To ensure a secure account binding, it is essential that each customer is able to control and pre-approve the ACME account keys that are authorized to request certificates against their account, particularly for certificate types that have billing implications, or that have additional verification requirements beyond what can be done inline within the ACME protocol, such as OV [[BR]], EV [[EVG]], Qualified Website [[QWAC]], or Verified Mark Certificates [[VMC]].
```

Reply:
I´d remove the "particularly for certificate types that have billing implications" because IMO that´s dependant on the CA if the´d like to charge for it but it´s not part of a technical implementation.